docs/install.md

```diff
@@ -65,6 +65,22 @@ make
 make install
 ```
 
+##Arch
+Install dependencies:
+```sh
```

Comment:
sudo is safer to use in general, and you seldom see `su root` in installation documents. sudo is most commonly used I think.